src/Stripe.net/Services/Sources/StripeSourceService.cs

```diff
@@ -40,6 +40,19 @@ public virtual StripeSource Detach(string customerId, string sourceId, StripeReq
             );
         }
 
+        public virtual StripeList<StripeSource> List(string customerId, StripeSourceListOptions listOptions = null, StripeRequestOptions requestOptions = null)
+        {
+            var url = string.Format(Urls.CustomerSources, customerId);
```

Comment:
Is it worth using a method to share the code? Not sure but that's what we do in other services (though the URL is used more than twice)